### DISCUSSION TITLE
Cannot access free GitHub Team

### DISCUSSION THREAD

**Discussion by xinpl** [community/community#55276] [GitHub Education]
### Hi everyone,

Hello,

I couldn't access the free GitHub Team feature for a teacher. This is similar to a previous thread: https://github.com/orgs/community/discussions/50702

The solution in the thread doesn't work for me. I cannot find this button.

Whenever I choose to upgrade my current organization, it shows me the regular options with payments. Could you help me with this?

Thanks!
-Xin

**Comment by samus-aran** [comment]
Hi @xinpl

Are you still experiencing this issue? Have you been successful in upgrading your organisation to the team plan?

**Reply by xinpl** [reply]
No, there is no problem now. It just needed a bit of time.

Thanks!
Xin
